python/plugins/db_manager/db_plugins/spatialite/connector.py

```diff
@@ -571,18 +571,33 @@ def runVacuum(self):
         c.execute('VACUUM')
         self.connection.isolation_level = '' # reset to default isolation
 
+    def commentTable(self, schema, tablename, comment):
+        """Comment the table"""
+        return ''
+
+    def commentTable(self, schema, tablename, comment):
+        return ''
+
+    def unCommentTable(self, schema, tablename):
+        return ''
+
+    def getComment(self, tab, field, db):
+        """Returns the comment for a field"""
+        return ''
+
     def addTableColumn(self, table, field_def):
         """ add a column to table """
         sql = u"ALTER TABLE %s ADD %s" % (self.quoteId(table), field_def)
+        self._execute_and_commit(sql)
```

Comment:
What's this for?

Reply:
Same here, i wrote it again cause a bad merge deleted it . I don't know what it is for

Reply:
Undo!

Reply:
Well in fact i don't know where it comes from.. Dropped